src/sass/rx/rx.scss

```diff
@@ -90,6 +90,7 @@ button.usa-button-outline {
 
 .rx-loading-error {
   font-weight: bold;
+  width: 66%;
```

Comment:
Is this something you want on mobile as well? It's possible to make it 100% on mobile and then 66% on everything else, if you want.